utility_network/display-utility-associations/src/main/java/com/esri/samples/display_utility_associations/DisplayUtilityAssociationsSample.java

```diff
@@ -233,7 +234,7 @@ private ImageView addSymbolToImageView(SimpleLineSymbol symbol) {
     
     // add the connectivity symbol to the image view to display it in the legend
     ImageView imageView = new ImageView();
-    ListenableFuture<Image> connectivityImage = symbol.createSwatchAsync(0x00000000);
+    ListenableFuture<Image> connectivityImage = symbol.createSwatchAsync(Color.BLACK);
```

Comment:
In this case, `0x00000000` was transparent, as the first 2 digits indicate 00 opacity - by setting it to `Color.BLACK` the sample looks like this:
![image](https://user-images.githubusercontent.com/56556822/191220820-13a879bb-eb99-428c-a7af-3580591ebca1.png)

So in this case, I'd swap that out to be `Color.TRANSPARENT`, which better matches the previous color.